public/usage-examples/utilities/index_of/index_of-1-find-word.cpp

```diff
@@ -0,0 +1,27 @@
+#include "splashkit.h"
+
+int main()
+{
+    // Get sentence input from the user
```

Comment:
The comment here on line 5 and the one on line 9 do not exist in the other language files

Reply:
I've just updated this